jinahub/indexers/searcher/HnswlibSearcher/hnswlib_searcher.py

```diff
@@ -160,7 +160,9 @@ def index(
         if docs is None:
             return
 
-        docs_to_update = docs.traverse_flat(traversal_paths)
+        docs_to_update = docs.traverse_flat(
```

Comment:
As agreed, make this a parameter to control whether or not this filter is applied